### PR TITLE
EWM-373: Add sticky header for seed groups in account selector

### PR DESCRIPTION
**Pull request by levitckii-daniil**
## Description

- Enables users to always see which seed group they are viewing when scrolling through accounts
- Improves navigation efficiency in the account selection screen

## Solution

Refactored the `SelectAccountWidget` to use `CustomScrollView` with `SliverPersistentHeader` for each seed group. Each seed header remains pinned to the top while scrolling through its accounts.

Key changes:
- Replaced `SingleChildScrollView` with `CustomScrollView` using slivers
- Created `_SeedItemStickyDelegate` to handle the sticky header behavior
- Implemented `_SeedItemSliverSection` to manage each seed group as a sliver section
- Headers remain visible and interactive while scrolling

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 Documentation update
- [ ] 🧹 Code refactoring

## Testing Instructions

1. Navigate to the account selection screen
2. Create or have multiple seed phrases with multiple accounts
3. Scroll through the account list
4. Verify that seed headers stick to the top when scrolling
5. Verify that clicking on headers still expands/collapses account lists
6. Test with both expanded and collapsed states

## QA Testing Results

- [ ] All testing instructions have been followed
- [ ] Feature works as expected
- [ ] No regressions found
- [ ] Edge cases tested

### CODE REVIEW

File: lib/feature/wallet/widgets/select_account/select_account_widget.dart [hunk 1]
```diff
@@ -36,33 +36,32 @@ class SelectAccountWidget
...
+              if (list == null) return const SizedBox.shrink();
+
+              final slivers = <Widget>[];
+
+              for (var index = 0; index < list.length; index++) {
+                final data = list[index];
+                final shouldAutoExpand = data.hasCurrentAccount(currentAccount);
+
+                // Add each seed as a sticky section
+                slivers.add(
```
Comment (knightsforce) [inline lib/feature/wallet/widgets/select_account/select_account_widget.dart:49]: Нам точно нужно вычислять в build? По-хорошему проверки делать 1 раз где-то не здесь.

Comment (levitckii-daniil) [reply lib/feature/wallet/widgets/select_account/select_account_widget.dart:49]: Так это по сути .map или тот-же List.generate, который был раньше. Для его вычисления нужны результаты DoubleSourceBuilder из-за чего в другое место его никак не вынести

**Review by knightsforce — APPROVED** [review]
